tests/e2e/workloads/bdi/test_bdi_workload.py

```diff
@@ -0,0 +1,59 @@
+import ocs_ci.ocs.bdi.config as bdi_config
+from ocs_ci.ocs.bdi.bdi_base_class import TestBdiWorkloadBaseClass
+
+# from ocs_ci.framework.pytest_customization.marks import (
+#     skipif_ceph_capacity_less_equal_1T,
+# )
+
+
+class TestBdiWorkloadSF1(TestBdiWorkloadBaseClass):
+    """
+    Tests BDI workload on SF (Scale Factor) = 1
+
+    """
+
+    def test_bdi_workload(self):
+        self.run()
+
+
+class TestBdiWorkloadSF10(TestBdiWorkloadBaseClass):
+    """
+    Tests BDI workload on SF (Scale Factor) = 10
+
+    """
+
+    def test_bdi_workload(self):
+        bdi_config.pvc_size = "200Gi"
+        bdi_config.scale_factor = 10
+        bdi_config.configure_timeout = 2400
+        bdi_config.data_load_timeout = 3600
+        self.run()
+
+
+class TestBdiWorkloadSF100(TestBdiWorkloadBaseClass):
```

Comment:
do we need to run with all the 4 SFs? Just for a single test to run this can be too much and it increases the test run duration a lot and I don't see a point running with all the 4 SFs. Can we please run with one SF?

Reply:
Removed all other SF tests, left only with SF=10

Reply:
Removed all other SF tests, left only with SF=10